src/hotspot/share/gc/shenandoah/shenandoahGenerationalHeap.cpp

```diff
@@ -839,7 +839,6 @@ class ShenandoahGenerationalUpdateHeapRefsTask : public WorkerTask {
             region_progress = true;
```

Comment:
I don't think we need this `region_progress` variable anymore.

Reply:
Good catch... Removed.